recipes/nb_conda_kernels/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.0.3" %}
+{% set gh_org = "Anaconda-Platform" %}
+{% set gh_repo = "nb_conda_kernels" %}
+
+package:
+  name: {{ gh_repo }}
+  version: {{ version }}
+
+source:
+  fn: {{ gh_repo }}-{{ version }}.tar.gz
+  url: https://github.com/{{ gh_org }}/{{ gh_repo }}/archive/{{ version }}.tar.gz
+  sha256: 6cdc08c3337c06e5cffacce150dbd10f87334489768e4c8804e045d28f65532a
+
+build:
+  number: 1
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - notebook >=4.2.0
+    - python
+
+test:
+  requires:
+    - coverage
+    - flake8
+    - nodejs
+    - nose
+    - notebook >=4.2.0
+    - python
+    - requests
+    - mock  # [py2k]
```

Comment:
Though `mock` has continued to develop independently and changes have periodically been merged back into Python `master`.